Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo

```diff
@@ -46,8 +46,7 @@ package RobotR3
     connect(axis.flange, load.flange_a)
```

Comment:
Change group="startAngles" to group="Start angles" and group="endAngles" to group="End angles"?

Reply:
And SI.Conversions.NonSIunits.Angle_deg again?

Reply:
Fixed, thanks.